test/bdd/features/didexchange_e2e.feature

```diff
@@ -18,4 +18,5 @@ Feature: Decentralized Identifier(DID) exchange between the agents
     And   "Bob" receives invitation from "Alice"
     And   "Alice" waits for post state event "completed"
     And   "Bob" waits for post state event "completed"
-
+    And   "Alice" retrieve connection record and validate state is "completed"
```

Comment:
retrieves connection record and validates that connection state is "completed"

Reply:
same as Alice

Reply:
done